### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by marcus-herrmann**
Concerns #8

### CODE REVIEW

**Review by radibit — APPROVED** [review]

**Comment by josefie** [comment]
Great! There is only one thing that doesn't really make sense on our own page, as there is no list of supporters there. And we are not listed there anyway:
> You can find a list of organizers to contact for each of the supporters of this code of conduct at the bottom of this page.

So maybe we should add some contact information somewhere (I wouldn't count twitter in this case)?

File: code-of-conduct.html [hunk 1]
```diff
@@ -1,5 +1,93 @@
...
+---
+layout: default
+title: Code of conduct
 description: Code of conduct for the A11y Meetup Berlin
----
\ No newline at end of file
+---
+
+<p><strong>Out Code of conduct is based on the <a
+        href="http://berlincodeofconduct.org/">berlincodeofconduct.org</a></strong>.</p>
```
Comment (justmarkup) [inline code-of-conduct.html]: One small typo, it should be Our instead of Out here

**Comment by marcus-herrmann** [comment]
@justmarkup Fixed
@josefie Agree on that paragraph, deleted it for now. A list of supporters / a general contact option could be a topic for an issue on its own, I guess.

**Review by justmarkup — APPROVED** [review]

**Comment by justmarkup** [comment]
@josefie I would like to merge this PR, but at the same time keep the issue https://github.com/a11yberlin/a11y-meetup-berlin.de/issues/8 open, because this is a good start here but we should do more.

**Review by josefie — APPROVED** [review]

**Pull request merged**